assets/css/edd-admin.css

```diff
@@ -2431,6 +2420,92 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div#post-body-content {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: 1fr;
+	grid-gap: 20px;
+	max-width: 1000px;
+}
+
+.postbox [class^="edd-export"] {
```

Comment:
Smart!

`.postbox` could be updated to `.edd-reports-export` if you make the above change as well. It shouldn't be breaking anything as is, but a more specific selector will be less likely to break things in the future.

Reply:
I just learned about that wildcard selector today!